cesium/channel_test.go

```diff
@@ -74,6 +75,31 @@ var _ = Describe("Channel", func() {
 					),
 				)
 			})
+			Describe("Opening db on existing folder", func() {
+				It("Should not panic when opening a db in a directory with already existing files", func() {
+					s := MustSucceed(fs.Sub("sub"))
+					f := MustSucceed(s.Open("dir1", os.O_CREATE))
+					Expect(f.Close()).To(Succeed())
+
+					db, err := cesium.Open("", cesium.WithFS(s))
+					Expect(err).ToNot(HaveOccurred())
+					Expect(db.Close()).To(Succeed())
+				})
+
+				It("Should turn parsable numeric folders into channels", func() {
+					s := MustSucceed(fs.Sub("sub"))
+					_, err := s.Sub("1")
```

Comment:
This is kind of concerning. This DB has no meta or index file, so we don't know the data type of the channel (along with other critical info). What will happen if we try to open a reader on this directory? I feel like this case should fail

Reply:
The DB will create the meta file during `openUnaryOrVirtual` and will create `index.domain` and `counter.domain` files while opening the unary and domain DBs. So I think this is fine. If those files are corrupted, then I feel like DB creation should still succeed and internal errors should reflect those errors

Reply:
NVM. On a second thought it doesn't make sense to create a meta file if we don't know the meta properties of that channel. I will make a check in `ReadOrCreate` meta that makes it fail if channel's properties are not set.

Reply:
This has been resolved?